src/App.tsx

```diff
@@ -14,7 +23,9 @@ export const App = () => {
       </div>
 
       <div className="sidebar">
-        <FindMovie />
+        <FindMovie
+          onAddMovie={addMovie}
+        />
```

Comment:
just Wrapping if you have more than 2 attributes or length > 80
```suggestion
        <FindMovie onAddMovie={addMovie} />
```